core/src/main/java/jenkins/model/ModelObjectWithContextMenu.java

```diff
@@ -128,6 +128,19 @@ public ContextMenu add(String url, String icon, String text, boolean post, boole
             return this;
         }
 
+        /** @since 2.87 */
```

Comment:
It may take us a while to merge something, so the safer approach is to `@since TODO` and let this be inserted on merge. Better TODO than an altogether wrong version because someone missed it.